Libraries/WebSocket/RCTReconnectingWebSocket.m

```diff
@@ -46,7 +67,7 @@ static void my_os_log_error_impl(void *dso, os_log_t log, os_log_type_t type, co
   }
 }
 
-#endif /* __IPHONE_11_0 */
+#endif /* RCT_DEV */
```

Comment:
This change is incorrect. This endif belongs to its previous `__IPHONE_11_0` check

Reply:
Correct, changed back.